NorthstarDLL/CMakeLists.txt

```diff
@@ -85,6 +85,7 @@ add_library(NorthstarDLL SHARED
             "scripts/client/scriptbrowserhooks.cpp"
             "scripts/client/scriptmainmenupromos.cpp"
             "scripts/client/scriptmodmenu.cpp"
+			"scripts/client/scriptoriginauth.cpp"
```

Comment:
Tabs

Reply:
Resolved in [f16c11d](https://github.com/R2Northstar/NorthstarLauncher/pull/468/commits/f16c11db8e86f7f46a9cd1886840105d8bbb1dfe)